### PR TITLE
Enterprise create kafka interactive (#1819)

### PR DESCRIPTION
**Pull request by jackdelahunt**
* refactor: moving list enterprise clusters to util package

* feat: start of interactive enterprise create

Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: pkg/shared/remote/DynamicServiceConstants.go [hunk 1]
```diff
@@ -19,4 +19,5 @@ type AmsConfig struct {
 	TrialProductQuotaID         string `json:"trialQuotaProductId"`
 	LongLivedQuotaProductID     string `json:"longLivedQuotaProductId"`
 	ResourceName                string `json:"resourceName"`
+	EnterpriseProductQuotaID    string `json:"RHOSAKCC"`
```
Comment (machi1990) [inline pkg/shared/remote/DynamicServiceConstants.go:22]:
```suggestion
	EnterpriseProductQuotaID    string `json:"enterpriseProductQuotaID"`
```

Comment (machi1990) [reply pkg/shared/remote/DynamicServiceConstants.go:22]: This also needs to be added here https://github.com/redhat-developer/app-services-cli/blob/b5cbad899af6c9ae215a2401f463888747039b16/pkg/shared/remote/service-constants.json#L4

File: pkg/shared/accountmgmtutil/ams.go [hunk 2]
```diff
@@ -106,12 +107,17 @@ func GetOrgQuotas(f *factory.Factory, spec *remote.AmsConfig) (*OrgQuotas, error
 				case "RHOSAKEval":
 					remainingQuota := int(quota.GetAllowed() - quota.GetConsumed())
 					evalQuotas = append(evalQuotas, QuotaSpec{QuotaEvalType, remainingQuota, quotaResource.BillingModel, quota.CloudAccounts})
+				//	this isn't working as it should be as the product is hardcoded here
+				case "RHOSAKCC":
```
Comment (machi1990) [inline pkg/shared/accountmgmtutil/ams.go:111]:
```suggestion
				case spec.EnterpriseProductQuotaID:
```

File: pkg/shared/accountmgmtutil/ams.go [hunk 3]
```diff
@@ -87,7 +88,7 @@ func GetOrgQuotas(f *factory.Factory, spec *remote.AmsConfig) (*OrgQuotas, error
 		return nil, err
 	}
 
-	var standardQuotas, marketplaceQuotas, trialQuotas, evalQuotas []QuotaSpec
+	var standardQuotas, marketplaceQuotas, trialQuotas, evalQuotas, enterpriseQuotas []QuotaSpec
```
Comment (machi1990) [inline pkg/shared/accountmgmtutil/ams.go:91]: this should be refactored later on and base of the logic by the billing model information that's returned by KFM for each supported instance type

File: pkg/shared/remote/DynamicServiceConstants.go (hunk 1 above)
Comment (dimakis) [reply pkg/shared/remote/DynamicServiceConstants.go:22]: got this on the other PR will close this one. thanks for the great catch 👍

File: pkg/shared/accountmgmtutil/ams.go (hunk 3 above)
Comment (dimakis) [reply pkg/shared/accountmgmtutil/ams.go:91]: reflected in the code as a comment and noted 👍

**Pull request closed without merging**